week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
+    
+}
```

전체에 대하여 css를 적용할 때, 전체를 감싸는 새로운 div를 만드는 대신 이미 존재하는 body 태그를 활용할 수 있습니다.
```suggestion
body {
    width: 650px;
    margin: 0 auto;
    padding-top: 90px;
    text-align: center;
    background-color: #c1bcea;
    color: white;
}
```